packages/tools/devtools/devtools-example/src/ClientUtilities.ts

```diff
@@ -4,105 +4,104 @@
+	CollaborativeDevtoolExampleObjectsContainerRuntimeFactory,
+	ICollaborativeDevtoolExampleAppModel,
+} from "./Container";
 
 /**
  * This module contains Fluid Client utilities, including Container creation / loading.
  */
 
-/**
- * Type returned from when creating / loading the Container.
- */
-export interface ContainerLoadResult {
-	container: IFluidContainer;
-	services: TinyliciousContainerServices;
-}
-
 /**
  * Basic information about the container, as well as the associated audience.
  */
 export interface ContainerInfo {
 	/**
-	 * The initialized Fluid Container.
+	 * The initialized Container.
 	 */
-	container: IFluidContainer;
+	container: IContainer;
 
 	/**
 	 * The Container's unique ID. Also referred to as the "Document ID".
 	 */
 	containerId: string;
+
+	/**
+	 * Model object for the Container.
+	 */
+	obj: CollaborativeDevtoolExampleAppObjects;
 }
 
-function initializeTinyliciousClient(logger?: ITelemetryBaseLogger): TinyliciousClient {
-	console.log(`Initializing Tinylicious client...`);
-	return new TinyliciousClient({
+/**
+ * Creates a new Fluid Container from the provided client and container schema.
+ */
+export function createLoader(
+	logger?: ITelemetryBaseLogger,
+): SessionStorageModelLoader<ICollaborativeDevtoolExampleAppModel> {
+	const codeLoader = new StaticCodeLoader(
+		new CollaborativeDevtoolExampleObjectsContainerRuntimeFactory(),
+	);
+	const loader = new SessionStorageModelLoader<ICollaborativeDevtoolExampleAppModel>(
+		codeLoader,
 		logger,
-	});
+	);
+
+	return loader;
+}
+
+/**
+ * Creates a new Fluid Container from the provided client and container schema.
+ */
+async function createContainer(
+	loader: SessionStorageModelLoader<ICollaborativeDevtoolExampleAppModel>,
+): Promise<ContainerInfo> {
+	const createResponse = await loader.createDetached("1.0");
+	const containerId = await createResponse.attach();
+	const container = createResponse.model.container;
+	return { container, containerId, obj: createResponse.model.collaborativeObj };
+}
+
+/**
+ * Loads an existing Container for the given ID.
+ */
+async function loadContainer(
+	loader: SessionStorageModelLoader<ICollaborativeDevtoolExampleAppModel>,
+	id: string,
+): Promise<ContainerInfo> {
+	const model = await loader.loadExisting(id);
+	const container = model.container;
+	return { container, containerId: id, obj: model.collaborativeObj };
 }
 
 /**
  * Creates a new Fluid Container from the provided client and container schema.
  *
- * @param containerSchema - Schema with which to create the container.
+ * @param loader - The Loader instance to use for creating the Container.
  * @param setContentsPreAttach - (optional) Callback for setting initial content state on the
  * container *before* it is attached.
- * @param logger - (optional) Telemetry logger to provide to client initialization.
  *
  * @throws If container creation or attaching fails for any reason.
  */
 export async function createFluidContainer(
```

Comment:
I don't think we need these functions anymore. They basically just wrap calls to the new `createContainer` and `loadContainer` functions you added above.

Reply:
Merge these functions